content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -90,45 +92,45 @@ request rate doesn't affect the memory consumption.
 
 ### Latency
 
-Since Istio injects a sidecar proxy on the data path, latency is an important
+Since Istio injects a sidecar proxy or ztunnel proxy on the data path, latency is an important
```

Comment:
we don't inject ztunnel proxy... consider reword it to the request traverse through sidecar or ztunnel or/and waypoint proxy....